### PR TITLE
Clarify the differences between `gfx`, `gfx-hal`, and `wgpu` in the README

### PR DESCRIPTION
**Pull request by termhn**
https://www.reddit.com/r/rust_gamedev/comments/b01zy9/please_help_me_understand_the_gfxrs_architecture/

Several others have been confused about this too in the past

PR checklist:
- [ ] `make` succeeds (on *nix)
- [ ] `make reftests` succeeds
- [ ] tested examples with the following backends:
- [ ] `rustfmt` run on changed code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ gfx-rs is a low-level, cross-platform graphics abstraction library in Rust. It c
   * [OpenGL 2.1+/ES2+](src/backend/gl)
 * `gfx-warden` which is a data-driven reference test framework, used to verify consistency across all graphics backends.
 
+## The `gfx` crate
+
+This repository was originally home to the `gfx` crate, which is now deprecated. You can find the latest versions of the code for that crate in the `pre-ll` branch of this repository.
```
Comment (kvark) [inline README.md]: make `pre-ll` a link?

Comment (kvark) [inline README.md]: make `gfx` a link?

File: README.md [hunk 2]
```diff
@@ -27,6 +27,14 @@ gfx-rs is a low-level, cross-platform graphics abstraction library in Rust. It c
...
+
+The master branch of this repository is now focused on developing `gfx-hal` and its associated backend and helper libraries, as described above. `gfx-hal` is a complete rewrite of `gfx`, but it is not necessarily the direct successor to `gfx`. Instead, it serves a different purpose than the original `gfx` crate, by being 'lower level' than the original. Hence, the name of `gfx-hal` was originally `ll`, which stands for 'lower level', and the original `gfx` is now referred to as `pre-ll`.
```
Comment (kvark) [inline README.md]: make `gfx-hal` a link to crates?

Comment (kvark) [inline README.md]: let's put "lower level" into double quotes, unlike the code

**Comment by termhn** [comment]
👍

**Pull request merged**